packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -1,6 +1,10 @@
 import 'whatwg-fetch';
 import { CallApi } from '../types';
 
+const cacheAvailable = 'caches' in self;
```

Comment:
I'd `UPPER_CASE` this since it's used as a constant in this file

Reply:
+1